### PR TITLE
Allow for the customization of the kvm section in domains

### PR DESCRIPTION
**Pull request by Libadoxon**
Fixes [19](https://github.com/AshleyYakeley/NixVirt/issues/19).

Things done:

- Add kvm section in `generate-xml/domain.nix`
- Add new check, called `template-windows-3`, for verifying the newly created kvm section
- Update Changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 * Lib:
     * XML generation:
         * Allow QEMU `commandline` in domains
+        * Allow in feature section, customization of the `kvm` options, in domains
```
Comment (AshleyYakeley) [inline CHANGELOG.md:8]: v0.4.2 has already been released, so please create a new "Unreleased" section.

**Comment by AshleyYakeley** [comment]
(Everything else looks good.)

**Comment by Libadoxon** [comment]
Fixed

**Comment by AshleyYakeley** [comment]
Thanks!

**Pull request merged**